config.toml

```diff
@@ -99,8 +99,19 @@ trusting_period = '14days'
 # which have changed between two blocks.
 # Default: { numerator = '1', denominator = '3' }, ie. 1/3.
 trust_threshold = { numerator = '1', denominator = '3' }
-[chains.filters]
-channels = [['transfer', 'channel-0']]
+
+# Channel filter specification. Default: no filter
+# This whole section can be deleted and then all filtering is disabled for this chain.
+[chains.channel_filter]
+# Channel filter type:
+# 	- 'allow': permit relaying only on the port/channel id in the list below,
+#	- 'deny': permit relaying on any channel except for the list below.
+type = 'deny'
+# List of ports and channels.
+list = [
+	['transfer', 'channel-0'],
```

Comment:
I think the example filter list should be given as commented code. Otherwise people just copying the example config.toml will just have their packets not forwarded by Hermes when following the tutorial.

Reply:
I agree!